src/unix/newlib/mod.rs

```diff
@@ -598,10 +618,19 @@ pub const NI_NAMEREQD: ::c_int = 4;
 pub const NI_NUMERICSERV: ::c_int = 0;
 pub const NI_DGRAM: ::c_int = 0;
 
-pub const EAI_FAMILY: ::c_int = -303;
-pub const EAI_MEMORY: ::c_int = -304;
-pub const EAI_NONAME: ::c_int = -305;
-pub const EAI_SOCKTYPE: ::c_int = -307;
+cfg_if! {
+    if #[cfg(target_os = "vita")] {
+        pub const EAI_FAMILY: ::c_int = -6;
+        pub const EAI_MEMORY: ::c_int = -10;
+        pub const EAI_NONAME: ::c_int = -2;
```

Comment:
The ordering here doesn't really make sense after this patch... maybe this could be `pub use vita::{EAI_NONAME, ...};` ?

Reply:
I like that idea, I'll change it

Reply:
Just did it

We can't add `pub use` as it goes against the library styling ([pub use must go after all constants](https://github.com/rust-lang/libc/blob/main/ci/style.rs)), but there's already a `pub use self::vita::*` at the end.